### PR TITLE
update mojave spec

### PR DESCRIPTION
**Pull request by CommanderK5**
Add flutter to pre-installed soft.

### CODE REVIEW

File: macos-mojave_5c6d3bfa2c7d3a66e32eaf2e.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ Installed versions:
 - 2.3.7 (system)
 - 2.5.1
 
+## Flutter
+
+- v1.2.1-stable (macOS)
```
Comment (markoa) [inline macos-mojave_5c6d3bfa2c7d3a66e32eaf2e.md]: @CommanderK5 is `macOS` mentioned here because there's an Android edition?

Comment (CommanderK5) [reply macos-mojave_5c6d3bfa2c7d3a66e32eaf2e.md]: @markoa there are version for Windows and Linux, not sure why it ended up here. I can remove it if you don't mind.

Comment (markoa) [reply macos-mojave_5c6d3bfa2c7d3a66e32eaf2e.md]: It just made me wonder what it means, may happen to others too, so 👍 for removing it.

**Comment by shiroyasha** [comment]
@CommanderK5 @markoa is this version ready to be merged?

**Pull request merged**